src/content/docs/de/guides/deploy.mdx

```diff
@@ -41,9 +42,23 @@ Manche Hosting-Anbieter stellen eine eigene Kommandozeilen&shy;schnittstelle (CL
 
 1. Installiere die CLI deines Hosting-Anbieters global, zum Beispiel:
 
-    ```bash
+<PackageManagerTabs>
+    <Fragment slot="npm">
+    ```shell
     npm install --global netlify-cli
     ```
+    </Fragment>
+    <Fragment slot="pnpm">
+    ```shell
+    pnpm add --global netlify-cli
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn global add netlify-cli
+    ```
+    </Fragment>
+  </PackageManagerTabs>
```

Comment:
An example of the correct indenting for lists:
```suggestion
   <PackageManagerTabs>
     <Fragment slot="npm">
     ```shell
     npm install --global netlify-cli
     ```
     </Fragment>
     <Fragment slot="pnpm">
     ```shell
     pnpm add --global netlify-cli
     ```
     </Fragment>
     <Fragment slot="yarn">
     ```shell
     yarn global add netlify-cli
     ```
     </Fragment>
   </PackageManagerTabs>
```
As the example shows, there are 3 spaces, matching where the text from item 1 begins. This might be different on other examples, but the formatting of anything being indented by 2 should be the same.